### ISSUE TITLE
Publish kgctl binaries for every linux, macos, and windows with every release

### ISSUE THREAD

**Issue by mrhein** [squat/kilo#56]
feature request:
kgctl under releases for linux/macosx/win

meanwhile:
kgctl as docker image => https://hub.docker.com/r/mrhein/kgctl

**Comment by Ehco1996** [comment]
try to build kgctl in mac 

found this err


```
# github.com/containernetworking/plugins/pkg/ip
vendor/github.com/containernetworking/plugins/pkg/ip/addr.go:40:40: undefined: netlink.FAMILY_ALL
vendor/github.com/containernetworking/plugins/pkg/ip/addr.go:51:19: undefined: syscall.IFA_F_TENTATIVE
vendor/github.com/containernetworking/plugins/pkg/ip/addr.go:51:43: undefined: syscall.IFA_F_DADFAILED
# github.com/squat/kilo/pkg/iproute
pkg/iproute/iproute.go:51:39: undefined: netlink.FAMILY_ALL
pkg/iproute/iproute.go:69:9: undefined: netlink.AddrReplace
pkg/iproute/iproute.go:78:39: undefined: netlink.FAMILY_ALL
# github.com/squat/kilo/pkg/route
pkg/route/route.go:47:13: undefined: netlink.RouteReplace
pkg/route/route.go:53:42: undefined: netlink.FAMILY_ALL
pkg/route/route.go:64:12: undefined: netlink.RuleAdd
pkg/route/route.go:70:18: undefined: netlink.RuleList
pkg/route/route.go:70:35: undefined: netlink.FAMILY_ALL
pkg/route/route.go:76:13: undefined: netlink.RuleDel
pkg/route/route.go:96:12: undefined: netlink.RouteSubscribe
pkg/route/route.go:110:9: undefined: unix.RTM_DELROUTE
pkg/route/route.go:195:48: undefined: netlink.FAMILY_ALL
pkg/route/route.go:203:24: undefined: netlink.RuleList
pkg/route/route.go:203:24: too many errors
```

**Comment by AntonOfTheWoods** [comment]
So we really have to build `kgctl` ourselves???

**Comment by squat** [comment]
I just merged support for compiling kgctl for Windows and mac OS using the Go toolchain in #92. The next step is to set up GitHub actions to automatically upload binaries when a release is published :)

**Comment by squat** [comment]
This is now implemented by #111 thanks to @leonnicolas! The next release will automatically have binaries for every platform uploaded

**Comment by leonnicolas** [comment]
Binaries are now live in the [0.2.0 Release](https://github.com/squat/kilo/releases/tag/0.2.0) of Kilo!
